ceph-salt-formula/salt/ceph-salt/time-sync.sls

```diff
@@ -1,5 +1,7 @@
 {% if pillar['ceph-salt']['time_server'].get('enabled', True) %}
 {% set time_server = pillar['ceph-salt']['time_server']['server_host'] %}
+{% set time_server_short_name = time_server.split('.', 1)[0] %}
+{% set time_server_is_minion = time_server_short_name in pillar['ceph-salt']['minions']['all'] %}
```

Comment:
@ricardoasmarques I guess we should wait until #210 is fixed, right? Then we will be able do:

    {%  set time_server_is_minion = time_server in pillar['ceph-salt']['minions']['all'] %}

Reply:
@smithfarm Yes, here is the PR https://github.com/ceph/ceph-salt/pull/211